tests/utils/jestUtils.js

```diff
@@ -5,9 +5,29 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 // via https://github.com/salesforce/lwc/blob/81a90709bb6eb84b41f1cfe29d6afe15c9a97a13/scripts/jest/utils/index.ts
-import fs from 'fs'
-import path from 'path'
-import glob from 'glob'
+import fs from 'node:fs'
+import path from 'node:path'
+
+// Based on https://github.com/fs-utils/fs-readdir-recursive/blob/f810b44/index.js
+// Can be replaced with fs.readdirSync(..., { recursive: true }) when we drop Node 18 support
+function readdirRecursiveSync (root, files, prefix) {
+  prefix = prefix || ''
+  files = files || []
+
+  const dir = path.join(root, prefix)
+  if (fs.existsSync(dir)) {
+    if (prefix) {
+      files.push(prefix)
+    }
+    if (fs.statSync(dir).isDirectory()) {
+      for (const name of fs.readdirSync(dir)) {
+        readdirRecursiveSync(root, files, path.join(prefix, name))
+      }
+    }
+  }
+
+  return files
+}
```

Comment:
`glob` and its various cousins (e.g. `fast-glob`) were causing too many problems due to transitive dependencies breaking due to ESM vs CJS. So I got rid of `glob` entirely.